packages/lodestar/src/api/rest/routes/validator/publishAttestation.ts

```diff
@@ -30,18 +31,24 @@ export const registerAttestationPublishEndpoint = (fastify: IFastifyServer, modu
     "/attestation",
     opts,
     async (request, reply) => {
+      let statusCode = 200;
       try {
-        await modules.opPool.attestations.receive(
-          fromJson<Attestation>(
-            request.body.attestation,
-            modules.config.types.Attestation
-          )
+        const attestation = fromJson<Attestation>(
+          request.body.attestation,
+          modules.config.types.Attestation
         );
+        if(!await modules.opPool.attestations.verifyAndReceive(
+          modules.chain.latestState, attestation
+        )) {
+          statusCode = 202;
+          const attestationHash = hashTreeRoot(attestation, modules.config.types.Attestation);
+          modules.logger.error(`Cannot verify attestation ${attestationHash.toString("hex")}`);
+        }
```

Comment:
I've asked in eth discord whether we need to gossip attestation in case it failed validation

Reply:
thanks, please let me know when this is clear